README.md

```diff
@@ -61,30 +61,31 @@ decimal by 1). Otherwise, the integration may complain of a duplicate unique ID.
+| `icons`                          | bool                | **Optional** | Whether to show icons instead of text labels                                                                                                                    | `false`             |
+| `num_segments`                   | number              | **Optional** | Number of forecast segments to show (integer >= 1)                                                                                                              | `12`                |
+| ~~`num_hours`~~                  | number              | **Optional** | _Deprecated:_ Use `num_segments` instead                                                                                                                        | `12`                |
+| `offset`                         | number              | **Optional** | Number of forecast segments to offset from start                                                                                                                | `0`                 |
+| `label_spacing`                  | number              | **Optional** | Space between time/temperature labels (integer >= 1)                                                                                                            | `2`                 |
+| `colors`                         | [object][color]     | **Optional** | Set colors for all or some conditions                                                                                                                           |                     |
+| `hide_hours`                     | bool                | **Optional** | Whether to hide hour labels under the bar                                                                                                                       | `false`             |
+| `hide_temperatures`              | bool                | **Optional** | Whether to hide temperatures under the bar                                                                                                                      | `false`             |
+| `round_temperatures`             | bool                | **Optional** | Whether to round temperatures to the nearest whole number                                                                                                       | `false`             |
+| `hide_bar`                       | bool                | **Optional** | Whether to hide the bar itself                                                                                                                                  | `false`             |
+| `icon_fill`                      | [string][icon_fill] | **Optional** | Whether to repeat the icon inside the bar                                                                                                                       | `'single`           |
```

Comment:
This the only real change in this file. Everything else is adding spaces to keep columns aligned.